### PR TITLE
fix bug in file type selection used in add table query

### PR DESCRIPTION
**Pull request by jenrogers1207**
There is a very initial table that loads the rows for a selected table as well, but the main reason for this is the bug fix.

### CODE REVIEW

File: client/src/components/FileInput.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default {
       let fileName = file.name.split('.')
       let extension = fileName[fileName.length - 1]
       for(let type in this.types){
-        if(this.types[type].includes(extension)){
+        if(this.types[type].extension.includes(extension)){
           return type
```
Comment (waxlamp) [inline client/src/components/FileInput.vue]: I think this whole section could possibly be simplified using JavaScript's `some()` and `endswidth()` methods. Something like:

```javascript
if (this.types[type].extension.some(ext => extension.endswith(ext))) {
  return type;
}
```

but it's not urgent to make this change.

File: client/src/views/TableDetail.vue [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
   <div>
     <h1>Table: {{`${this.workspace}/${this.table}`}}</h1>
+
+<!-- 
     <ul>
-      <li v-for="field in fields" :key="field">{{field}}</li>
+     <li v-for="field in fields" :key="field">{{field}}</li> 
+       <li v-for="row in rowKeys" :key="row[0].value">{{row[0].value}}</li>
     </ul>
+-->
```
Comment (waxlamp) [inline client/src/views/TableDetail.vue]: If we're really dumping this code that's been commented out, I'd prefer if we actually cut it out altogether.

File: client/src/views/TableDetail.vue [hunk 3]
```diff
@@ -15,18 +23,28 @@ export default {
   props: ['workspace', 'table'],
   data () {
     return {
-      fields: []
+      //fields: []
```
Comment (waxlamp) [inline client/src/views/TableDetail.vue]: Same thing -- let's cut the code out if it shouldn't be here.

**Comment by waxlamp** [comment]
If the table work isn't complete yet, it's totally fine to do a PR just to fix that bug. Let me know if you'd like to split the bugfix out from the rest, or if you'd still like to merge this as-is.

**Comment by jenrogers1207** [comment]
Removed commented code. I'm thinking we can merge as-is and then I'll create a separate branch for the table improvement. I'm still currently styling this!

**Pull request merged**